torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5543,6 +5543,19 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_conversion(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device)
+
+    cases = ((),
+             (2, 3))
+    memory_format_options = [None, torch.preserve_format, torch.contiguous_format]
+
+    def generator():
+        for shape, memory_format in itertools.product(cases, memory_format_options):
+            yield(SampleInput(make_arg(shape, requires_grad=requires_grad),
```

Comment:
Given that we unconditionally set `requires_grad` grad here, we can merge it into the `partial` above.